.github/dependabot.yaml

```diff
@@ -1,10 +1,11 @@
 version: 2
 updates:
-  - package-ecosystem: "yarn"
+  - package-ecosystem: "npm"

```

Comment:
Why change to NPM if Aurora Client also uses yarn?

Reply:
`npm` is just how dependabot calls the ecosystem that uses a `package.json`, so this still work if the repo uses yarn.